dex/src/main/scala/com/wavesplatform/dex/cli/Actions.scala

```diff
@@ -0,0 +1,574 @@
+import monix.execution.{ExecutionModel, Scheduler}
+import monix.execution.schedulers.SchedulerService
+import sttp.client3._
+
+import java.io.PrintWriter
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.util.{Base64, Scanner}
+import java.util.concurrent.atomic.AtomicLong
+import scala.concurrent.{Await, TimeoutException}
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try, Using}
+
+object Actions {
+
+  sealed trait SeedFormat
+
+  private[cli] object SeedFormat {
+
+    case object RawString extends SeedFormat
+    case object Base64 extends SeedFormat
+    case object Base58 extends SeedFormat
+
+    implicit val seedFormatRead: scopt.Read[SeedFormat] = scopt.Read.reads {
+      case "raw-string" => RawString
+      case "base64" => Base64
+      case "base58" => Base58
+      case x => throw new IllegalArgumentException(s"Expected 'raw-string', 'base64' or 'base58', but got '$x'")
+    }
+
+  }
+
+  private val backend = HttpURLConnectionBackend()
+
+  // noinspection ScalaStyle
+  def deleteOrderBook(args: Args): Unit = {
+    cli.log(
+      s"""
+         |Passed arguments:
+         |  DEX config path : ${args.configPath}
+         |  DEX REST API    : ${args.dexRestApi}
+         |  Timeout         : ${args.timeout}
+         |  Asset Pair      : ${args.assetPair}
+         |Running in background
+         |""".stripMargin
+    )
+
+    implicit val scheduler: SchedulerService = Scheduler.singleThread(
+      name = "time-impl",
+      daemonic = true,
+      executionModel = ExecutionModel.AlwaysAsyncExecution
+    )
+
+    val key = readSecretFromStdIn("Enter X-API-KEY: ")
+
+    val currentOffset = sendRequest(s"${args.dexRestApi}/matcher/debug/currentOffset", key).toInt
+    sendRequest(s"${args.dexRestApi}/matcher/orderbook/${args.assetPair.replace('-', '/')}", key, "delete")
+    sendRequest(s"${args.dexRestApi}/matcher/debug/currentOffset", key, "post")
+
+    val validation = Task(sendRequest(args.dexRestApi, "oldestSnapshotOffset", key).toInt <= currentOffset)
+      .delayExecution(1.second)
+      .onErrorRestart(Long.MaxValue)
+      .restartUntil(_ == true)
+      .timeout(args.timeout)
+      .runToFuture
+
+    Try(Await.ready(validation, args.timeout)) match {
+      case Success(_) => println(s"Success!")
+      case Failure(e) =>
+        e match {
+          case _: TimeoutException => println("Snapshots wasn't saved before reaching timeout")
+          case _ => println(s"Other error happened:\n${e.printStackTrace()}")
+        }
+        System.exit(1)
+    }
+  }
+
+  // noinspection ScalaStyle
+  def generateAccountSeed(args: Args): Unit = {
+    val seedPromptText = s"Enter the${if (args.accountNonce.isEmpty) " seed of DEX's account" else " base seed"}: "
+    val rawSeed = readSeedFromFromStdIn(seedPromptText, args.seedFormat)
+    val accountSeed = KeyPair(args.accountNonce.fold(rawSeed)(AccountStorage.getAccountSeed(rawSeed, _)))
```

Comment:
`val seedPromptText = s"Enter the${if (args.accountNonce.isEmpty) " seed of DEX's account" else " base seed"}: "
    val rawSeed = readSeedFromFromStdIn(seedPromptText, args.seedFormat)
    val accountSeed = KeyPair(args.accountNonce.fold(rawSeed)(AccountStorage.getAccountSeed(rawSeed, _)))`

duplicated lines in createAccountStorage method
can be move tho method